src/Sulu/Bundle/MediaBundle/Resources/js/containers/MultiMediaDropzone/MultiMediaDropzone.js

```diff
@@ -80,12 +84,18 @@ class MultiMediaDropzone extends React.Component<Props> {
         });
 
         return Promise.all(uploadPromises).then((...media) => {
-            this.props.onUpload(...media);
+            onUpload(...media);
 
             setTimeout(() => {
-                this.props.onClose();
+                onClose();
                 this.destroyMediaUploadStores();
             }, 1000);
+        }).catch((e) => {
+            setTimeout(() => {
+                onClose();
```

Comment:
when not closing the popover, the "upload items" will not be removed therefore still be present if the popover is closed and opened a second time. but you are right about the error message too, it is not very visible while the popover is openend

Reply:
I would say if it would just have been about the uploaded items it would have to be solved in a different way, but since the error message should be visible it's probably better this way anyway.